src/workloads/c2c/mongosync_helpers.py

```diff
@@ -0,0 +1,117 @@
+import argparse
```

Comment:
I like this movement to python. I wonder about how formal we want to be with the script? It's a one-off here. Should it be a module? That is likely overkill, but I wanted to say it out loud.